### PR TITLE
cart page

### PR DESCRIPTION
**Pull request by aryehlev**
alot of code was taken from @OdedHellman 's PR i just figured we should get it PR'd

### CODE REVIEW

**Review by simzacks — CHANGES_REQUESTED** [review]
remove the log file.
It says there are conflicts. Please pull and rebase and submit again

File: superx/routes/home.py [hunk 1]
```diff
@@ -1,18 +1,46 @@
...
+                    "price": same_item.price
+                })
+                total_prices['shufersal']['price'] += same_item.price
+
+            elif same_item.chain_id == sd['victory']['chain_id']:
+                total_prices['victory']['list'].append({
+                    "name": item['name'],
+                    "price": same_item.price
+                })
+                total_prices['victory']['price'] += same_item.price
```
Comment (shaibrown5) [inline superx/routes/home.py:41]: you are doing the same thing in every if, the only thing that changes is the name of the super. mabe try this (easier on the eyes)

```
for same_item in price_list_of_item:
    super_name = ''
    
    if same_item.chain_id == sd['mega']['chain_id']:
        super_name = 'mega'
    elif same_item.chain_id == sd['shufersal']['chain_id']:
        super_name = 'shufersal'
    elif same_item.chain_id == sd['victory']['chain_id']:
        super_name = 'victory'

    total_prices[super_name]['list'].append({
        "name": item['name'],
        "price": same_item.price
    })
    total_prices[super_name]['price'] += same_item.price
```

Comment (aryehlev) [reply superx/routes/home.py:41]: good point, done!

**Comment by aryehlev** [comment]
ok i have updated it, there are no conflicts now

File: superx/routes/home.py (hunk 1 above)
Comment (shaibrown5) [reply superx/routes/home.py:41]: dont forget to push the code

**Comment by OdedHellman** [comment]
Nicely done @aryehlev... looks good!

**Pull request closed without merging**